README.md

```diff
@@ -72,7 +76,9 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+[:friends].each do
```

Comment:
`friends` is a normal Ruby variable that contains an array. No need to convert it to a symbol.

```rb
friends.each do |friend|
```